dev/create-release/releaseutils.py

```diff
@@ -149,7 +152,11 @@ def get_commits(tag):
             if not is_valid_author(author):
                 author = github_username
         # Guard against special characters
-        author = unidecode.unidecode(unicode(author, "UTF-8")).strip()
+        try:               # Python 2
+            author = unicode(author, "UTF-8")
+        except NameError:  # Python 3
+            author = str(author)
+        author = unidecode.unidecode(author).strip()
```

Comment:
could you just put this above:
```
if sys.version > '3':
    unicode = str
```
and then just do?
```
author = unidecode.unidecode(unicode(author)).strip()
```

I don't think you need to specify "UTF-8" because either way it will be a unicode object

Reply:
@BryanCutler I think we do need to specify it because it won't be unicode type in Python 2, we get these from run_cmd which call's Popen communicate which returns a regular string.

Reply:
Yes... My other worry in Py2 would be if [__sys.setdefaultencoding()__](https://docs.python.org/2/library/sys.html#sys.setdefaultencoding) was set to somthing other that utf-8.  That method was also thankfully dropped in Py3.

Reply:
My thought was that we are first casting `author` this to unicode already with `unicode(author)` and it doesn't really matter if it is "UTF-8" or not because we then immediately decode it into ASCII with `unidecode`, which can handle it even it it wasn't "UTF-8", so the end result should be the same I believe.  It was just to clean up a little, so not a big deal either way.  The way it is now replicates the old behavior, so it's probably safer.

Reply:
> The way it is now [...] it's probably safer.

Let's agree to leave this as is in this PR.  EOL of Python 2 in 500 daze away so safe is better.